src/StoreKeeper/WooCommerce/B2C/Imports/ProductImport.php

```diff
@@ -1160,13 +1163,23 @@ public function setConfigurableAttributes(&$newProduct, $product, $optionsConfig
      *
      * @throws WordpressException
      */
-    private function createProductVariations($parentProduct, $optionsConfig, $log_data)
+    private function syncProductVariations($parentProduct, $optionsConfig, $log_data)
     {
         // Sorting?
         $data_store = $parentProduct->get_data_store();
         $data_store->sort_all_product_variations($parentProduct->get_id());
 
         $associatedShopProducts = $optionsConfig->get('configurable_associated_shop_products', false);
+        $backofficeVariationStorekeeperIds = array_column($associatedShopProducts, 'shop_product_id');
+
+        $variationPostIds = $parentProduct->get_children();
+        // Check if id no longer exist in the associated products and delete the post object
+        foreach ($variationPostIds as $variationPostId) {
+            $variationStorekeeperId = $this->getPostMeta($variationPostId, 'storekeeper_id', false);
```

Comment:
what with `$variationStorekeeperId === false` case? Or will it be reenabled in the later loop?